### DISCUSSION TITLE
How to get out of this Gamification shizzle?

### DISCUSSION THREAD

**Discussion by nextlevelshit** [community/community#28651] [Other Feature Feedback, Questions, & Ideas]
I want to code and not another social network ... Can I turn this stuff somehow off? I want git, not a family nor friends nor followers.

**Comment by melvincwng** [comment]
@nextlevelshit , follow these steps below to turn off the achievements on your GitHub Profile:

1. Go to 'Settings'
2. Scroll down to 'Profile Settings'
3. Uncheck the option 'Show Achievements on my profile'
4. This will prevent your achievements from being displayed on your profile
5. For more information, you can refer to this documentation [link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile) over here.

Hope this helps.

Thanks, and take care 😄

**Comment by jfullstackdev** [comment]
and you can even make your profile private, 

go to Settings then check it

![image](https://user-images.githubusercontent.com/47092464/183346592-1ae1738d-d227-4f0f-b5cb-c97fb77d0e82.png)

**Comment by thenithinbalaji** [comment]
I don't think turning it off is a solution. 
The very existence of achievement badges gamifies the entire Github.

Read this
https://github.com/community/community/discussions/28686#discussioncomment-3347349
